mongo/src/main/java/com/redhat/lightblue/mongo/crud/BasicDocSaver.java

```diff
@@ -284,6 +288,7 @@ private void updateDocs(CRUDOperationContext ctx,
                             ctx.getFactory().getInterceptors().callInterceptors(InterceptPoint.PRE_CRUD_UPDATE_DOC, ctx, doc.inputDoc);
                             merge.merge(doc.oldDoc, doc.newDoc);
                             Translator.populateDocHiddenFields(doc.newDoc, md);
+                            MongoSafeUpdateProtocol.overwriteDocVer(doc.newDoc,docver);
```

Comment:
Maybe this should append w/ timeout clean up like other updates, to prevent removing an a doc version of a successful update that happened immediately prior to this save.

1. Update (A) partial success, needs to check for concurrent mods
2. Save (B) retrieves updates from A, but overwrites doc ver
3. Check for concurrent mods for A has false positive where overlaps with B, because docver is gone even though both are successes